trunk/examples/witness-checking/regression/correctness/apache-get-tag.i-witness.yaml

```diff
@@ -0,0 +1,54 @@
+-
+  entry_type: loop_invariant
+  metadata:
+    format_version: 0.1
+    uuid: "072e87df-48b4-476f-ac99-67b3aa304c45"
+    creation_time: "2023-08-02T17:01:58+02:00"
+    producer:
+      name: UNUSED
+      version: UNUSED
+    task:
+      input_files:
+        - "C:\ultimate\trunk\examples\witness-checking\regression\correctness\apache-get-tag.i"
```

Comment:
do we need absolute paths here? And wouldnt that lead to problems during testing in CI or on other machines?

Reply:
We don't use any of the metadata for validation. I just used the filename from CDT parser here. For our regression tests wr could use some dummy string, it does not really matter.